### PR TITLE
Remove most content.

### PR DESCRIPTION
**Pull request by jdswinbank**
There was a lot of material duplicated between this page and the Stingray
documentation. This was doubly confusing, because that information was (in
genereal) only relevant to Stingray itself, and not to HENDRICS and DAVE which
are also covered by this page.

This PR makes this page into just a “landing page” for the StingraySoftware
organization. It provides links to all the associated projects; their pages
are assumed to provide all the information that readers will need, so it isn't
duplicated here.

(The one thing that wasn't already in the Stingray docs was the list of talks,
so I'm going to PR an addition of that there.)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,54 +28,15 @@ <h2 id="project_tagline">Next-generation spectral-timing software</h2>
...
-<h4><a id="presentations" class="anchor" href="#presentations" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Presentations</h4>
-<p><ul>
-    <li>(upcoming) AAS 17th High-Energy Astrophysics Division meeting 2019</li>
-    <li><a alt="Stingray EWASS2018" href="http://ascl.net/wordpress/2018/05/24/software-in-astronomy-symposium-presentations-part-3/">European Week of Astronomy and Space Science 2018</a></li>
-    <li><a alt="Stingray HEAD17" href="https://speakerdeck.com/abigailstev/stingray-open-source-spectral-timing-software">AAS 16th High-Energy Astrophysics Division meeting 2017</a></li>
-    <li><a alt="Stingray EWASS17" href="http://ascl.net/wordpress/2017/07/23/special-session-on-and-about-software-at-ewass-2017/">European Week of Astronomy and Space Science 2017</a></li>
-    <li><a alt="Stingray PyAstro16" href="https://speakerdeck.com/abigailstev/stingray-pyastro16">Python in Astronomy 2016</a></li>
-</ul></p>
-
-<h4><a id="acknowledgements" class="anchor" href="#acknowledgements" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Acknowledgements</h4>
```
Comment (matteobachetti) [inline index.html:73]: I think the acknowledgements should stay, either here or maybe in the Stingray repository

**Review by matteobachetti — CHANGES_REQUESTED** [review]
Thanks a lot @jdswinbank, the edits make sense. Minor change requested.

File: index.html (hunk 1 above)
Comment (jdswinbank) [reply index.html:73]: They're listed (albeit without the graphics) in the Stingray docs: https://stingraysoftware.github.io/stingray/acknowledgements.html. @dhuppenkothen indicated she thought that was the best place for them. Should I add the logos there as well?

Comment (matteobachetti) [reply index.html:73]: Ah, ok. I think that Stingraysoftware benefited as a whole from PyCharm's free licenses and from GSoC, so it's maybe good to have them here too. @dhuppenkothen, @abigailStev, what do you think?

**Comment by jdswinbank** [comment]
Hey @matteobachetti — how's this?

**Review by matteobachetti — APPROVED** [review]

**Comment by matteobachetti** [comment]
Looks good!

**Comment by matteobachetti** [comment]
@jdswinbank if you're done, I'll merge

**Comment by jdswinbank** [comment]
Go for it! 👍

**Pull request merged**